tests/unit/legelhold.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# MinIO Python Library for Amazon S3 Compatible Cloud Storage,
+# (C) 2020 MinIO, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from unittest import TestCase
+
+from nose.tools import eq_
+
+from minio import xml
+from minio.legalhold import LegalHold
+
+
+class LegalHoldTest(TestCase):
+    def test_status(self):
+        config = LegalHold(True)
+        xml.marshal(config)
+
+        config = xml.unmarshal(
+            LegalHold,
+            """<LegalHold xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
+  <Status>OFF</Status>
+</LegalHold>""",
```

Comment:
xml indentation is off. Other than that LGTM

Reply:
@poornas What do you mean by xml indentation is off? why is it necessary for XML?